rsplan/primitives.py

```diff
@@ -307,9 +309,9 @@ def _interpolated(
         # straight segments) and dtheta (step size / turn radius) for curved segments.
         step = step_size if self.is_straight else step_size / self.turn_radius
```

Comment:
Do we still need to calculate step?

Reply:
Good catch, should be using that in the `num_steps` calc, fixing